input/pagecontent/general-requirements.md

```diff
@@ -410,7 +262,96 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
+  
+      Example: ExplanationOfBenefit resource where the patient's insurance coverage is not available.
+      ~~~
+      {
+        "resourceType" : "ExplanationOfBenefit",
+            ...
+            "outcome" : "complete",
+              "insurance" : [
+                {
+                  "focal" : true,
+                  "coverage" : {
+                    "extension" : [
+                      {
+                        "url" : "http://hl7.org/fhir/StructureDefinition/data-absent-reason",
+                        "valueCode" : "unknown"
+                      }
+                    ]
+                  }
+                }
+              ],
+              ...
+          }
+      ~~~
+      
+  - when the AU Core profile mandates a sub-element, such as a valid Identifier or Reference data type, then the resource must contain the sub-element otherwise the resource will not be conformant,
+
+1. For *coded* data elements:
+    - *required* binding strength (CodeableConcept or code datatypes):
+      - the appropriate "unknown" concept code **SHALL** be populated if available.
+      - For AU Core profiles, the following *mandatory* or *conditionally mandatory* status elements with required binding have no appropriate "unknown" concept code:
+        - `AllergyIntolerance.clinicalStatus`
+        - `Condition.clinicalStatus`
+        - `Immunization.status`
+    - *example*, *preferred*, or *extensible* binding strengths (CodeableConcept, or Coding datatypes):
+      - when the system has text but no coded value, only the text sub-element is populated.
+      - when there is neither text or coded value:
+        - the appropriate "unknown" concept code **SHALL** be populated if available when the binding strength is *extensible*
+        - when the value set does not have an appropriate "unknown" concept code, use `unknown` from the [DataAbsentReason Code System](http://terminology.hl7.org/CodeSystem/data-absent-reason).
+
+        Example: AllergyIntolerance resource where the manifestation is unknown.
+        ~~~
+        ...
+        "reaction" : [
+          {
+            "manifestation" : [
+              {
+                "coding" : [
+                  {
+                    "system" : "http://terminology.hl7.org/CodeSystem/data-absent-reason",
+                    "code" : "unknown",
+                    "display" : "unknown"
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+        ...
+        ~~~
+
+
+### Suppressed Data
+In some circumstances, specific pieces of data may be hidden due to security or privacy reasons. 
+
+When an element definition is *optional* (minimum cardinality = 0), including elements labelled *Must Support*, an
+- AU Core Responder **SHALL NOT** populate the element in the resource when a value is suppressed.  
+
+When an element definition is *Mandatory* (minimum cardinality > 0), 
```

Comment:
Optional and mandatory have already been defined in the content above, which I don't think is an issue. Maybe consider defining Mandatory in the same way (defined as both minimum cardinality =1 and minimum cardinality >0) for consistency. 
Same applies to Missing Data section as well.

Reply:
No change. The ballot approved text has been restored. This will be retained.